providers/efa/man/efadv_query_device.3.md

```diff
@@ -0,0 +1,63 @@
+---
+layout: page
+title: EFADV_QUERY_DEVICE
+section: 3
+tagline: Verbs
+date: 2019-04-22
+header: "EFA Direct Verbs Manual"
+footer: efa
+---
+
+# NAME
+
+efadv_query_device - Query device capabilities
+
+# SYNOPSIS
+
+```c
+#include <infiniband/efadv.h>
+
+int efadv_query_device(struct ibv_context *ibvctx,
+                       struct efadv_device_attr *attr);
+```
+
+# DESCRIPTION
+
+**efadv_query_device()** Queries EFA device specific attributes.
+
+Compatibility is handled using the comp_mask field which serves as both an
+indication for requested attributes and returned attributes.
+
+```c
+struct efadv_device_attr {
+	uint8_t version;
+	uint8_t reserved[7];
+	uint64_t flags;
+	uint64_t comp_mask;
+	uint32_t max_sq_wr;
+	uint32_t max_rq_wr;
+	uint16_t max_sq_sge;
+	uint16_t max_rq_sge;
+	uint16_t inline_buf_size;
+};
+```
+
+*comp_mask*
+:	Compatibility mask.
+
```

Comment:
A good man page would describe what each field is

Reply:
Done